src/lumigo_tracer/examples/threads.py

```diff
@@ -0,0 +1,25 @@
+import concurrent.futures
```

Comment:
Why do we need the example file? Looks like the user doesn't have to do anything special in order to use threads.